src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -885,7 +877,6 @@ mod tests {
                 .unwrap();
 
             assert!(got.is_some());
-            assert_eq!(got.unwrap().to_bits(), (0.4545469470152683f64).to_bits());
```

Comment:
Maybe we can simply change the expect value of the test case. It sounds reasonable to test whether or not its result is unchanged for an unchanged seed.

Reply:
@breeswish  It seems that once the implementation of `XorShiftRng`  changed, the value we use in unit tests needs to be changed. So maybe we should rewrite a more universal unit test that is independent of the implementation of `XorShiftRng`.

Reply:
@breeswish 
Ok, I will add this test. 
There is another question: should we try different seeds and expect the result's same probability is below a specific value.

Reply:
@iosmanthus Yes. Generally it will be locked by our lock file so that we don't need to update the unit test even if it is changed.

@H-ZeX This looks to be a good idea. Nice to have. Feel free to include it in a new PR.